src/core/Akka.NodeTestRunner/Sink.cs

```diff
@@ -84,7 +88,7 @@ public void Dispose()
     /// that this class produces, so do not remove or refactor it.
     /// </remarks>
     /// </summary>
-    public class SpecPass
+    public class SpecPass : MarshalByRefObject
```

Comment:
Could you explain this to me? Why `MarshalByRefObject` - is this part of XUnit2 or related to some other changes?